README.md

```diff
@@ -3,15 +3,109 @@
 Ralph is a toolbox for your learning analytics, it can be used as a:
 
 - **library**, to fetch learning events from various backends, (de)serialize or
-  convert them from various standard formats such as
-  [xAPI](https://adlnet.gov/projects/xapi/),
+    convert them from various standard formats such as
+    [xAPI](https://adlnet.gov/projects/xapi/), or
+    [openedx](https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_logs/index.html)
 - **command-line interface** (CLI), to build data pipelines the UNIX-way™️,
 - **HTTP API server**, to collect xAPI statements (learning events)
   following the [ADL LRS
   standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree).
 
 ## Quick start guide
 
+### Testing the LRS server with Docker compose
+
+> [`curl`](https://curl.se) and [`jq`](https://stedolan.github.io/jq/) are
+> required to run some commands of this tutorial.
+
+To bootstrap a test environment on your machine, clone this project first and
+run the `bootstrap` Makefile target:
+
+```
+$ make bootstrap
```

Comment:
As `make bootstrap` starts the Elasticsearch container that is used afterwards, could we add the note about the `virtual memory limits` from https://github.com/openfun/ralph/blob/master/docs/contribute.md?plain=1#L72-L80 ?

Reply:
Good point. I've fixed that!